src/HotelPosSystem/BookingForm.cs

```diff
@@ -0,0 +1,123 @@
+﻿using HotelPosSystem.Entities;
+
+using Microsoft.EntityFrameworkCore;
+
+namespace HotelPosSystem {
+    internal class BookingForm {
+        private readonly BookingList _bookingList;
+
+        private ComboBox? _customerDropdown;
+        private DateTimePicker? _startDatePicker;
+        private DateTimePicker? _endDatePicker;
+        private ComboBox? _roomDropdown;
+        private TextBox? _commentTextBox;
+        private CheckBox? _paidForCheckBox;
+        private CheckBox? _checkedInCheckBox;
+
+        internal FlowLayoutPanel ContainerPanel;
+
+        internal BookingForm(BookingList bookingList) {
+            _bookingList = bookingList;
+            ContainerPanel = CreateFormControls();
+        }
+
+        internal FlowLayoutPanel CreateFormControls() {
```

Comment:
```suggestion
        private FlowLayoutPanel CreateFormControls() {
```
Om man använder denna klass borde man endast behöva anropa constructorn, ej denna metod